test/spmd/tensor/test_dtensor_ops.py

```diff
@@ -668,7 +669,15 @@ def wrapped(fn):
 }
 
 
+skip_bw = [None, "torch.isfinite"]
```

Comment:
could we add `tanhshrink`, `H`, `T`, and `sgn` to this list if that's the only ones that are failing?

Reply:
It was only for CPU ops so I was still trying to figure it out I guess :)
Added it to the skip_bw list for now to unblock the PR.

Reply:
The issue was that we had to remove xfail for the tanhshrink ops which got added back due to a bad merge. Removing the xfail for the tanhshrink and sgn ops means the test passes locally. Waiting for the green signal on circle CI.